### PR TITLE
Add new branch commit

### PR DESCRIPTION
**Pull request by MMGGYY66**
self review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (MMGGYY66) [inline index.html:25]: Please remove all unused codes from your project. This is because it makes your code unreadable.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+p {
+    color: blue;
+}
+
```
Comment (MMGGYY66) [inline css/style.css]: Please consider to leave 2 spaces as indentation in your style.css. To have a good clean code.

**Review by MMGGYY66 — COMMENTED** [review]
Changes required .
Please do those required changes below.

File: css/style.css (hunk 2 above)
Comment (abdulhamiid) [reply css/style.css]: You gave a polite suggestion and explained the reason you suggested the change 👍 .

File: index.html (hunk 1 above)
Comment (abdulhamiid) [reply index.html:25]: You made a review and left the comment under a line or block of code that fits the context the comment is about using the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request). 👍.

_**[OPTIONAL]**_ You can include emojis 🤗 and gifs to make your code review more human-friendly 👽.

**Review by abdulhamiid — APPROVED** [review]
# Status: :green_circle:  Approved! :partying_face:
Hi @MMGGYY66 , 

Your project is complete! Great job for nailing it right on the first attempt :1st_place_medal:. There is nothing else to say other than... it's time to merge it  :shipit:  Congratulations! :tada::100::star2:

giphy

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**